admin/helpers/logger.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const winston = require('winston')
+const config = require('../config')
+const morgan = require('morgan')
+
+let initialised
+
+const init = (app) => {
+  if (initialised) return
```

Comment:
Is that really required?
In which case would you expect `init` to be called again?

Reply:
that's the point - if it is called again (either accidentally or some misunderstands it), it won't bother re-adding all the transports